### DISCUSSION TITLE
Enable a hotkey to play a subject

### DISCUSSION THREAD

**Discussion by seanmiller26** [zooniverse/front-end-monorepo#6209] [Feature Requests]
## Package
_Choose from the list:_
- [ ] app-content-pages
- [ ] app-project
- [ ] lib-classifier
- [ ] lib-panoptes-js
- [ ] lib-react-components
- [ ] unknown

## Is your feature request related to a problem? Please describe.
Let's give volunteers the option of having a dedicated key that they can use to play/pause video/flipbook subjects.

## Describe the solution you'd like
The letter P which stands for play/pause, for instance.

**Comment by eatyourgreens** [comment]
FWIW, the video player uses space to play/pause, which is pretty standard for media players on the web. You do have to focus the player first, either by clicking on it or tabbing to it.

**Comment by eatyourgreens** [comment]
Here's a video of me using the video keyboard controls to play a video in Chrome. When the video player has keyboard focus:
- Space toggles play/pause.
- The cursor keys scrub backwards and forwards through a paused video.


https://github.com/zooniverse/front-end-monorepo/assets/59547/e8563bc0-db73-49ed-9b89-2b605ea7ed63

**Comment by eatyourgreens** [comment]
The space bar also toggles play/pause in the flipbook viewer, when the subject image is focused (tested on Daily Minor Planet.)

There is one difference from PFE. With PFE, I can click a subject to focus it. FEM subjects aren’t clickable like that. I have to tab to the subject image, which is inconvenient. 

Try it on [Daily Minor Planet here](https://www.zooniverse.org/projects/fulsdavid/the-daily-minor-planet/classify/workflow/22354). Tab through the page until the subject image is selected, and you'll find that you can toggle play/pause with the spacebar. There is one, big, level AA accessibility failure on that page: the subject image is focusable but has [no visible focus indicator](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html) in Firefox.

In Chrome, the focused image has a thin blue focus indicator, which would probably also fail. It's really easy to miss in this screenshot.

<img width="800" alt="Daily Minor Planet in Chrome, with the dark theme. The subject image has a thin blue focus indicator, easily lost against the black page background." src="https://github.com/zooniverse/front-end-monorepo/assets/59547/de28e274-5150-4a62-a2c5-0553956a3d92">

**Comment by eatyourgreens** [comment]
For keyboard and screen reader users, I would suggest:

- have skip links that take you from the top of the page to the classifier (or the classifier subject?)
- have clear focus indicators to indicate that the flipbook player or video player has focus.
- enable volunteers to select the player by clicking/tapping on it.
- communicate that both players support standard keyboard shortcuts that volunteers are probably already accustomed to from other sites (eg. YouTube?)
